tests/python/packages/isce3/geometry/radar_grid_spacing.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
+
+from collections.abc import Iterable
+
+import numpy as np
+import pytest
+
+import isce3
+from isce3.geometry import infer_radar_grid_spacing_from_geo_grid
+
+
+def make_linear_orbit(
+    initial_position: tuple[float, float, float],
+    velocity: tuple[float, float, float],
+    times: Iterable[float],
+    epoch: isce3.core.DateTime = isce3.core.DateTime("2000-01-01"),
+) -> isce3.core.Orbit:
+    """
+    Construct an `Orbit` representing a linear flight path with fixed velocity.
+
+    Parameters
+    ----------
+    initial_position : (float, float, float)
+        The position of the platform, in meters, in the Earth-centered, Earth-fixed
+        (ECEF) coordinate system at time t=0.
+    velocity : (float, float, float)
+        The velocity of the platform, in meters per second, in the same coordinate
+        system as `initial_position`.
+    times : iterable of float
+        The timepoints at which to sample the orbit state vectors, in seconds relative
+        to `epoch`.
```

Comment:
Can the times occur prior to the epoch? If not, what about:

```suggestion
        The timepoints at which to sample the orbit state vectors, in seconds since
        `epoch`.
```

Reply:
Negative time points are allowed, yes, but I think the suggestion is good anyway.